src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx

```diff
@@ -6,25 +6,29 @@ tags:
+* Modify alert conditions
+* Manage data retention settings  
+
+You can create custom roles and add any number of permissions to them. 
 
 To learn what permissions a role has, go to the user management UI and view a specific role. To find this UI: From <DNT>**[one.newrelic.com](https://one.newrelic.com/all-capabilities)**</DNT>, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings) in the lower right, and then go to: <DNT>**Administration > Access management > Roles**</DNT>.
 
-## What are permissions?  [#permissions-roles]
+## Important points about permissions [#important-points]
 
-A New Relic full platform user with no limitations (for example, a user in the <DNT>**Admin**</DNT> group) is able to use all features of the platform. Some of the things you can do in New Relic we've made available as <DNT>**permissions**</DNT>. You can add or remove these from a custom role, and we also use these permissions to differentiate between our [standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles). The permissions that we've made visible and available for selection are those we think you're most likely to find useful for common user management tasks.
+A New Relic full platform user with no limitations (for example, a user in the <DNT>**Admin**</DNT> group) is able to use all features of the platform. Many, but not all, of the tasks you can perform in New Relic are available as <DNT>permissions</DNT>. You can add or remove these from a custom role, and we also use these permissions to differentiate between our [standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles). The permissions that we've made visible and available for selection are those we think you're most likely to find useful for common user management tasks.
 
-There are a lot of New Relic functionalities that we <DNT>**don't**</DNT> make visible and available for selection. For example, there are various UI pages that you can access as any user, and that aren't gated by the permissions we expose. For another example: a user in a group with <DNT>**Organization**</DNT> [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings) can configure organization-level settings and that power is not available for adding to a custom role.
+There are a lot of New Relic functionalities that we don't make visible and available as permissions you could select. For example, there are various UI pages that you can access as any user and that aren't gated by the permissions we expose. For another example: A user in a group with <DNT>**Organization**</DNT> [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings) can configure organization-level settings and that power is not available for adding to a custom role.
```

Comment:
As vale says, is not > isn't

Reply:
@jeff-colucci sorry I missed your review. Could you explain this suggestion?